saleor/graphql/utils.py

```diff
@@ -14,6 +15,27 @@ def get_node(info, id, only_type=None):
     return node
 
 
+def get_nodes(ids, graphene_type):
+    """Return queryset of nodes of proper type."""
+    pks = []
+    for graphql_id in ids:
+        _type, _id = from_global_id(graphql_id)
+        assert str(graphene_type) == _type, (
+            'Must receive an {} id.').format(graphene_type._meta.name)
+        pks.append(_id)
+    nodes =  graphene_type._meta.model.objects.filter(pk__in=pks)
```

Comment:
Maybe cast it to a list explicitly here? Otherwise you're technically returning a QuerySet that could still be operated upon.

Reply:
Sure.